legend-engine-xt-relationalStore-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/executionPlan/nodes/SQLUpdateExecutionNode.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.protocol.pure.v1.model.executionPlan.nodes;
+
+
+public class SQLUpdateExecutionNode extends GenericSQLExecutionNode
```

Comment:
We should rename this class to better represent the way this node is been executed.  For example, this is not a a generic/simple update sql.  There are some functional considerations into the "update" when this node is executed, and a better name that highlight "not a simple sql update" will provide more context to the code.

Reply:
This has been renamed and split from sql execution node.